codecept.conf.js

```diff
@@ -66,6 +66,7 @@ exports.config = {
     addStatementOfServiceEventPage: './e2e/pages/events/addStatementOfServiceEvent.page.js',
     uploadC2DocumentsEventPage: './e2e/pages/events/uploadC2DocumentsEvent.page.js',
     draftStandardDirectionsEventPage: './e2e/pages/events/draftStandardDirectionsEvent.page.js',
+    draftCMOEventPage: './e2e/pages/events/draftCMOEvent.page.js',
```

Comment:
I don't like CMO, please use case management order

Reply:
changed it to draftCaseManagementOrderEventPage